pyproject.toml

```diff
@@ -67,6 +68,7 @@ swift-x-account-sharing = "swift_browser_ui.launcher:run_sharing"
 
 [project.urls]
 Source = "https://github.com/CSCfi/swift-browser-ui"
+Documentation = "https://swift-browser-ui.readthedocs.io"
```

Comment:
I didn't know about this ^^

Reply:
these get build from the docs folder on readthedocs, haven't been updated ... but it is the official external documentation link